raiden-dapp/src/views/SelectToken.vue

```diff
@@ -26,9 +22,14 @@
                   {{ token.symbol }} | {{ token.name }}
                 </v-list-tile-title>
                 <v-list-tile-sub-title class="token-address">
-                  {{ token.address }}
+                  {{ token.address | truncate }}
```

Comment:
There need to be ways to see the full address here.. again, you could truncate only if needed, or maybe put the address in a tooltip

Reply:
I will add a tooltip, thanks for pointing out.